swan-lake/development-tutorials/observability/supported-observability-tools-and-platforms/opensearch.md

```diff
@@ -0,0 +1,747 @@
+---
+title: Observe metrics, tracing and logs using OpenSearch
+description: See how Ballerina supports observability by exposing itself via metrics, tracing and logs to OpenSearch.
+keywords: ballerina, observability, metrics, tracing, logs, opensearch
+permalink: /learn/supported-observability-tools-and-platforms/opensearch/
+active: opensearch
+intro: Users can observe Ballerina programs with [OpenSearch](https://opensearch.org/), which is a community-driven, Apache 2.0-licensed open source search and analytics suite that makes it easy to ingest, search, visualize, and analyze data. It provides a highly scalable system for providing fast access and response to large volumes of data with an integrated visualization tool, OpenSearch Dashboards, that makes it easy for users to explore their data.
+---
+
+The sample [shop service](/learn/overview-of-ballerina-observability/#example-observe-a-ballerina-service) will be used in this guide.
+
+Follow the steps given below to view Ballerina metrics, tracing and logs in OpenSearch.
```

Comment:
```suggestion
Follow the steps given below to view Ballerina metrics, traces, and logs in OpenSearch.
```

Reply:
I am not 100% confident on this, pls double chekc.

Reply:
When I was searching in other websites, I noticed that both of them are used.

Reply:
Updated to `traces`